assets/kube-apiserver/kube-apiserver-deployment.yaml

```diff
@@ -250,35 +250,56 @@ spec:
       - name: portieris
         image: {{ .PortierisImage }}
         imagePullPolicy: IfNotPresent
+        command:
+          - "/portieris"
+        args:
+          - "--kubeconfig=/etc/openshift/kubeconfig/kubeconfig"
         ports:
         - containerPort: 8000
           name: http
           protocol: TCP
-        resources: {}
+{{ if .PortierisContainerResources }}
+        resources:{{ range .PortierisContainerResources }}{{ range .ResourceRequest }}
+          requests: {{ if .CPU }}
+            cpu: {{ .CPU }}{{ end }}{{ if .Memory }}
+            memory: {{ .Memory }}{{ end }}{{ end }}{{ range .ResourceLimit }}
+          limits: {{ if .CPU }}
+            cpu: {{ .CPU }}{{ end }}{{ if .Memory }}
+            memory: {{ .Memory }}{{ end }}{{ end }}{{ end }}
+{{ end }}
         terminationMessagePath: /dev/termination-log
         terminationMessagePolicy: File
+{{- if .PortierisLivenessProbe }}
+{{- $probe := .PortierisLivenessProbe }}
         livenessProbe:
           httpGet:
-            port: 8000
-            path: "/health/liveness"
-            scheme: HTTPS
-          initialDelaySeconds: 10
-          timeoutSeconds: 10
-        readinessProbe:
+            scheme: {{ or $probe.HttpGet.Scheme "HTTPS" }}
+            port: {{ or $probe.HttpGet.Port 8000 }}
+            path: {{ or $probe.HttpGet.Path "health/liveness" }}
```

Comment:
Sorry I missed this on my previous reviews, but the default should be `/health/liveness`.  The leading slash is missing.